GLTFSDK/Inc/GLTFSDK/Definitions.h

```diff
@@ -0,0 +1,10 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#pragma once
+
+#ifdef _WIN32
+#define GLTFSDK_CDECL __cdecl
+#else
+#define GLTFSDK_CDECL
+#endif

```

Comment:
I would recommend we do better than this. Instead of calling it `GLTFSDK_CDECL` we can instead call it `GLTFSDK_API` and allow people to override it to a different calling convention if they want. Check to see if `GLTFSDK_API` exists before setting the default.